src/main/java/study/step2/StringCalculator.java

```diff
@@ -0,0 +1,63 @@
+package study.step2;
+
+import java.util.Arrays;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StringCalculator {
+    private final String CUSTOM_DELIMITER_REGEX = "^//(.)\n";
```

Comment:
상수를 인스턴스 변수로 둬야 할 이유가 있을까요?

```suggestion
    private static final String CUSTOM_DELIMITER_REGEX = "^//(.)\n";
```